src/Framework/Table.php

```diff
@@ -0,0 +1,280 @@
+<?php
+namespace Give\Framework;
+
+use wpdb;
+
+/**
+ * Class Table
+ * @package Give\Database\Tables
+ *
+ * @since 2.9.0
+ */
```

Comment:
Since these are going in the Framework domain, please provide the class a thorough description that explains _when_ this should be used.

Reply:
fixed: https://github.com/impress-org/givewp/compare/3e779421a8ee...dff96c91fa5a